spec/features/manage_proposals/manage_proposal_view_spec.rb

```diff
@@ -4,31 +4,53 @@
     allow_any_instance_of(PermissionChecking).to receive(:is_non_nasa_draft_approver?).and_return(true)
   end
 
-# TODO: This test should function as is when the dummy data function is removed
-# from the controller.
-#  context 'when there are no approved proposals' do
-#    before do
-#      visit manage_proposals_path
-#    end
-#
-#    it 'has the appropriate text' do
-#      expect(page).to have_content('No Approved Proposals found')
-#    end
-#  end
-
-  context 'when using the dummy data' do
+  context 'when the user can be authenticated, has approver permissions, and there are two pages of proposals' do
     before do
+      set_as_proposal_mode_mmt
```

Comment:
Maybe monkey path the controller to shorten this test?

Reply:
More thinking doesn't really change my thoughts here; I think readability is more important than the ~10s that it could save us.